tests/unit/src/files/firebase-to-svelte-kit.test.js

```diff
@@ -44,67 +43,3 @@ test('convert string[] headers of any kind to csv string values', t => {
 
 	t.deepEqual(result, expected);
 });
-
-// Request
-test('firebase-functions.https.request GET is converted to SvelteKit Incoming request type correctly', t => {
-	const firebaseRequest = {
-		method: 'GET',
-		headers: {
-			'accept-language': 'en',
-			'set-cookie': ['some', 'cookie', 'data'],
-			host: 'us-central1-func.cloudfunctions.net',
-			'x-forwarded-proto': 'https',
-		},
-		url: '/url?some=thing',
-	};
-
-	const expectedKitRequest = {
-		method: 'GET',
-		headers: {
-			'accept-language': 'en',
-			'set-cookie': 'some,cookie,data',
-			host: 'us-central1-func.cloudfunctions.net',
-			'x-forwarded-proto': 'https',
-		},
-		rawBody: null,
-		host: 'https://us-central1-func.cloudfunctions.net',
-		path: '/url',
-		query: new URL('/url?some=thing', 'https://us-central1-func.cloudfunctions.net').searchParams,
-	};
-
-	const result = toSvelteKitRequest(firebaseRequest);
-
-	t.deepEqual(result, expectedKitRequest);
-});
-
-test('firebase-functions.https.request POST is converted to SvelteKit Incoming request type correctly', t => {
-	const firebaseRequest = {
-		method: 'POST',
-		headers: {
-			'accept-language': 'en',
-			'set-cookie': ['some', 'cookie', 'data'],
-			host: 'us-central1-func.cloudfunctions.net',
-			'x-forwarded-proto': 'https',
-		},
-		rawBody: Buffer.from('some-data', 'utf8'),
-		url: '/url?some=thing',
-	};
-
-	const expectedKitRequest = {
-		method: 'POST',
-		headers: {
-			'accept-language': 'en',
-			'set-cookie': 'some,cookie,data',
-			host: 'us-central1-func.cloudfunctions.net',
-			'x-forwarded-proto': 'https',
-		},
-		rawBody: Buffer.from('some-data', 'utf-8'),
-		host: 'https://us-central1-func.cloudfunctions.net',
-		path: '/url',
-		query: new URL('/url?some=thing', 'https://us-central1-func.cloudfunctions.net').searchParams,
-	};
-
-	const result = toSvelteKitRequest(firebaseRequest);
-
-	t.deepEqual(result, expectedKitRequest);
-});
```

Comment:
Why were these tests removed?

Reply:
I removed them because I couldn't come up with a suitable replacement. The new sveltekit library uses an Request object (different from the json provided) and I couldn't find a library to generate. 

I removed them because I considered the end to end tests a bit more valuable in terms of feedback but I do regret removing them nonetheless.

Reply:
Hmm, okay, I will have to check, because the Firebase API uses ExpressJS Request object which is different to the W3C, so we might still want to have a conversion function to capture some edge cases. I wanted to re-jig the integration tests too as they were a hack just to get something working for e2e tests while Kit API was changing rapidly.